src/index.js

```diff
@@ -53,45 +54,50 @@ function createBabelPluginFactory(customCallback = returnObject) {
 			externalHelpersWhitelist,
 			include,
 			runtimeHelpers,
+			transformGenerated,
 			...babelOptions
 		} = unpackOptions(pluginOptions);
 
+		if (transformGenerated) {
```

Comment:
I'm be curious for @Andarist's thoughts, but I wonder if this might be better exposed much clearly as really a totally different API, e.g.

```
import babel from "rollup-plugin-babel";

const plugin = babel.generated({ ... })
```
since all you do right now is branch the plugin code right up front anyway, and they could still share implementation logic. That would also allow for clearer documentation of the potentially for differing arguments between the two approaches.

Reply:
I actually like the idea. Will see if I can work it into the PR.

Reply:
I like it as well better than the previous version, good suggestion @loganfsmyth 👍